Tests/Command/CleanCommandTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/*
+ * This file is part of the FOSOAuthServerBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\OAuthServerBundle\Tests\Command;
+
+use FOS\OAuthServerBundle\Command\CleanCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CleanCommandTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \FOS\OAuthServerBundle\Command\CleanCommand
+     */
+    private $command;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|\Symfony\Component\DependencyInjection\ContainerInterface
+     */
+    private $container;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $command = new CleanCommand();
+
+        $application = new Application();
+        $application->add($command);
+
+        $this->container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+
+        $this->command = $application->find($command->getName());
+        $this->command->setContainer($this->container);
+    }
+
+    /**
+     * Delete expired tokens for classes that implement the TokenManagerInterface.
+     */
+    public function testItShouldRemoveExpiredTokensForTokenManagerInterfaces()
+    {
+        $expiredAccessTokens = 5;
+        $accessTokenManager = $this->getMock('FOS\OAuthServerBundle\Model\TokenManagerInterface');
+        $accessTokenManager
+            ->expects($this->once())
+            ->method('deleteExpired')
+            ->will($this->returnValue($expiredAccessTokens));
+
+        $expiredRefreshTokens = 3;
+        $refreshTokenManager = $this->getMock('FOS\OAuthServerBundle\Model\TokenManagerInterface');
+        $refreshTokenManager
+            ->expects($this->once())
+            ->method('deleteExpired')
+            ->will($this->returnValue($expiredRefreshTokens));
+
+        $expiredAuthCodes = 0;
+        $authCodeManager = $this->getMock('FOS\OAuthServerBundle\Model\TokenManagerInterface');
+        $authCodeManager
+            ->expects($this->once())
+            ->method('deleteExpired')
+            ->will($this->returnValue($expiredAuthCodes));
+
+        $containerMap = array(
+            'fos_oauth_server.access_token_manager' => $accessTokenManager,
+            'fos_oauth_server.refresh_token_manager' => $refreshTokenManager,
+            'fos_oauth_server.auth_code_manager' => $authCodeManager,
+        );
+
+        $this->container
```

Comment:
That is something I haven't thought of. For me a "unit" is just one class (and sometimes additional classes that are needed to run the tests), but since the container is basically a value object, it should be fine to use the real class. I will incorporate the changes in the next commit.